src/test/java/com/amashchenko/maven/plugin/gitflow/GitFlowFeatureStartMojoTest.java

```diff
@@ -0,0 +1,77 @@
+package com.amashchenko.maven.plugin.gitflow;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static org.testng.Assert.*;
+
+/**
+ * Created by krs on 2/18/17.
+ */
+public class GitFlowFeatureStartMojoTest {
```

Comment:
On closer look. This test as it is now is really don't needed, essentially it tests only `Pattern.matches` and not the logic in `GitFlowFeatureStartMojo`.